nestkernel/simulation_manager.cpp

```diff
@@ -375,6 +376,61 @@ nest::SimulationManager::simulate( Time const& t )
   t_slice_begin_ = timeval();
   t_slice_end_ = timeval();
```

Comment:
`t_slice_end_` is actually updated in `simulation_manager.cpp`, approx line 780 (master), by
```C++
gettimeofday( &t_slice_end_, NULL );
```
It is used to calculate the real-time factor for the progress display.